pkg/network/setup/netstat.go

```diff
@@ -181,8 +183,29 @@ func (c *NetStat) getPodInterfacefromFileCache(vmi *v1.VirtualMachineInstance, i
 	return podInterface, nil
 }
 
+func (c *NetStat) cleanVolatileCache(vmi *v1.VirtualMachineInstance) {
```

Comment:
The name does not hint what this cleans and without entering the function (and spending some time there to understand that it cleans interfaces marked for removal).